docs/root/configuration/operations/runtime.rst

```diff
@@ -273,7 +273,8 @@ The file system runtime provider emits some statistics in the *runtime.* namespa
   :widths: 1, 1, 2
 
   admin_overrides_active, Gauge, 1 if any admin overrides are active otherwise 0
-  deprecated_feature_use, Gauge, Number of times deprecated features were used. Detailed information about the feature used will be logged to warning logs in the form "Using deprecated option 'X' from file Y".
+  deprecated_feature_use, Counter, Total number of times deprecated features were used. Detailed information about the feature used will be logged to warning logs in the form "Using deprecated option 'X' from file Y".
+  deprecated_feature_active, Gauge, Number of times deprecated features were used. This is not carried over during hot restarts.
```

Comment:
Hm, do we actually want this to be an active counter (so if we had a config with deprecated config, then reloaded it out and replaced with non-deprecated we'd decrement as the bad config was no longer active) or should we come up with a name which indicates it's acting as a counter (deprecated_feature_use_since_process_start or some such)

Reply:
Good question -- let me look around to see if there's some consistency around naming these things (this is surely not the first gauge/counter of this nature -- "count since process start"... ).

Reply:
How about `deprecated_feature_seen_since_process_start` (since it might be used anymore)? Otherwise `deprecated_feature_use_since_process_start` works too, I think.